### PR TITLE
fix/project-preselection

### PR DESCRIPTION
**Pull request by manubo**

### CODE REVIEW

**Comment by manubo** [comment]
[BROWSER-EXTENSION: Fix preselected project and task](https://trello.com/c/V90VqsTL/7595-browser-extension-fix-preselected-project-and-task)

File: src/js/components/App.js [hunk 1]
```diff
@@ -68,21 +70,25 @@ class App extends Component {
   @observable formErrors = {}
 
   @computed get project() {
-    const { service, projects, lastProjectId } = this.props
+    const { service, projects, serviceLastProjectId, userLastProjectId } = this.props
+
+    console.log(serviceLastProjectId, service?.projectId, userLastProjectId)
```
Comment (pic) [inline src/js/components/App.js:75]: remove console log, right?

**Review by pic — APPROVED** [review]
@manubo looks good, just remove the console.log

File: src/js/components/App.js (hunk 1 above)
Comment (manubo) [reply src/js/components/App.js]: Yes, thanks. I removed it and merged.

**Pull request merged**